packages/screens/Settings/components/UserProfileModal.tsx

```diff
@@ -0,0 +1,325 @@
+  const marginChildrenVertical = layout.padding_x1;
+  const marginChildrenTop =
+    width < smallMobileWidth ? layout.padding_x1 : layout.padding_x2;
+
+  interface ProfileDataType {
+    id: number;
+    content: string;
+  }
+  const initProfileData: ProfileDataType[] = [
+    {
+      id: 1,
+      content: "Social Feed",
+    },
+    {
+      id: 2,
+      content: "Albums",
+    },
+    {
+      id: 3,
+      content: "NFTs",
+    },
+    {
+      id: 4,
+      content: "Activity",
+    },
+    {
+      id: 5,
+      content: "Succeed Quests",
+    },
+    {
+      id: 6,
+      content: "Pathwar Challenges",
+    },
+    {
+      id: 7,
+      content: "Gig Services",
+    },
+    {
+      id: 8,
+      content: "Governance Votes",
+    },
+    {
+      id: 9,
+      content: "Putted NFT to Rioters Footer",
+    },
+    {
+      id: 10,
+      content: "Shared servers",
+    },
+  ];
+
+  const [profileData, setProfileData] =
+    useState<ProfileDataType[]>(initProfileData);
+  const [confirmedData, setConfirmedData] = useState<any[]>([]);
+  const [hoveredIndex, setHoveredIndex] = useState<number>(0);
+
+  const styles = StyleSheet.create({
+    modalContainer: {
+      width,
+      height,
+      backgroundColor: "rgba(0, 0, 0, 0.9)",
+      textAlign: "center",
+    },
+    modalTitle: StyleSheet.flatten([
+      fontSemibold28,
+      {
+        marginTop:
+          width < mobileWidth
+            ? width < smallMobileWidth
+              ? 5 * layout.padding_x1
+              : 15 * layout.padding_x1
+            : 25 * layout.padding_x1,
+      },
+    ]),
+    modalText: StyleSheet.flatten([
+      fontMedium14,
+      {
+        marginHorizontal: "auto",
+        color: neutralA3,
+        marginTop: layout.padding_x1,
+        width: width < smallMobileWidth ? 0.9 * width : 15 * layout.padding_x4,
+      },
+    ]),
+    cardContainer: {
+      display: "flex",
+      flexDirection: "row",
+      width: "100%",
+      marginHorizontal: "auto",
+      justifyContent: "center",
+      alignItems: "center",
+      gap: layout.padding_x1_5,
+      marginTop:
+        width < mobileWidth
+          ? width < smallMobileWidth
+            ? 5 * layout.padding_x0_5
+            : 30 * layout.padding_x0_5
+          : 45 * layout.padding_x0_5,
+    },
+    unselectedCardBox: {
+      display: "flex",
+      flexDirection: "row",
+      justifyContent: "center",
+      alignItems: "center",
+      width: childrenWidth,
+      paddingVertical: layout.padding_x1_5,
+      borderColor: neutral33,
+      borderWidth: 1,
+      backgroundColor: neutral00,
+      borderRadius: layout.padding_x1,
+      position: "relative",
+    },
+    selectedCardBox: {
+      display: "flex",
+      flexDirection: "row",
+      justifyContent: "center",
+      alignItems: "center",
+      width: childrenWidth,
+      paddingVertical: layout.padding_x1_5,
+      borderColor: neutral33,
+      borderWidth: 1,
+      backgroundColor: neutral17,
```

Comment:
Between this line and `unselectedCardBox`, only the color changes, make the styles load what is common to them then override (or set) the color.